data/crac-creation/crac-creator-csa-profiles/src/main/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/CsaProfileCrac.java

```diff
@@ -87,15 +87,51 @@ public PropertyBags getShuntCompensatorModifications() {
     }
 
     public PropertyBags getTapPositionAction() {
-        return this.queryTripleStore(CsaProfileConstants.TAP_POSITION_ACTION);
+        return this.queryTripleStore(CsaProfileConstants.TAP_POSITION_ACTION, tripleStoreCsaProfileCrac.contextNames());
     }
 
     public PropertyBags getStaticPropertyRanges() {
-        return this.queryTripleStore(CsaProfileConstants.STATIC_PROPERTY_RANGE);
+        return this.queryTripleStore(CsaProfileConstants.STATIC_PROPERTY_RANGE, tripleStoreCsaProfileCrac.contextNames());
```

Comment:
You need to use the "queryTripleStore" method with contextNames list only if your sparql request contains the line "GRAPH <%s> {", where %s will be replaced by each context name, to execute request on each file of the original data zip. If the sparql request doesn't contain this %s, execute request with each context name will execute exactly the same request, on the whole data, as many times as the number of files in the zip, so it will be unnecessary longer.